gui/src/handle_args/HandleArgs.cpp

```diff
@@ -0,0 +1,81 @@
+/*
+** EPITECH PROJECT, 2024
+** Zappy [WSL: kali-linux]
+** File description:
+** HandleArgs
+*/
+
+#include "HandleArgs.hpp"
+#include <string>
+#include <sstream>
+#include <vector>
+#include <cctype>
+#include <cstdlib>
+#include <iostream>
+
+using namespace GUI;
+
+int HandleArgs::checkArgs(int nbArgs, char **args)
+{
+    try {
+        if (nbArgs == 2 &&
+            (std::string(args[1]) == "-help" || std::string(args[1]) == "-h")) {
```

Comment:
It's an error in the subject, I saw it with Hugo, and it shall have two '-' which result in '--help' and '-h' optionaly